packages/core/rollup.config.js

```diff
@@ -5,14 +5,18 @@ import json from '@rollup/plugin-json'
 import sveltePreprocess from 'svelte-preprocess'
 import typescript from '@rollup/plugin-typescript'
 import copy from '@rollup-extras/plugin-copy'
+import commonjs from '@rollup/plugin-commonjs'
 
 const production = !process.env.ROLLUP_WATCH
+const specificPackages = ['idna-uts46-hx'] // Replace with the package names you want to target
```

Comment:
Remove this comment?

Reply:
Was able to remove completely with the latest version of viem!